phpunit.xml.dist

```diff
@@ -2,13 +2,26 @@
 <phpunit bootstrap="./vendor/autoload.php"
          colors="true"
          convertNoticesToExceptions="false">
-         
+
     <testsuites>
         <testsuite name="Goetas XSD Reader Test Suite">
             <directory>./tests/</directory>
         </testsuite>
     </testsuites>
-    
+    <logging>
+        <log
+            type="coverage-html"
+            target="phpunit"
+            logIncompleteSkipped="true"
+        />
+        <log
```

Comment:
will this enable code coverage for all runs? (the previous setup was to run code coverage only on demand and only for a specific version)

Reply:
in terms of automated services, it won't make a difference- that'd require a clover logger.

This is more for local coverage checking- you get an html dashboard for reviewing the code coverage before pushing commits up.